diff --git a/src/AppBundle/Entity/CategoryRepository.php b/src/AppBundle/Entity/CategoryRepository.php
index edbff28..299c0d8 100644
--- a/src/AppBundle/Entity/CategoryRepository.php
+++ b/src/AppBundle/Entity/CategoryRepository.php
@@ -17,7 +17,6 @@ class CategoryRepository extends EntityRepository
         $dql = 'SELECT cat FROM AppBundle\Entity\Category cat ORDER BY cat.name DESC';
 
         $query = $this->getEntityManager()->createQuery($dql);
-        var_dump($query->getSQL());die;
 
         return $query->execute();
     }
