diff --git a/src/AppBundle/Entity/CategoryRepository.php b/src/AppBundle/Entity/CategoryRepository.php
index c6dc01e..4859fd7 100644
--- a/src/AppBundle/Entity/CategoryRepository.php
+++ b/src/AppBundle/Entity/CategoryRepository.php
@@ -17,6 +17,7 @@ class CategoryRepository extends EntityRepository
         $dql = 'SELECT cat FROM AppBundle\Entity\Category cat ORDER BY cat.name ASC';
 
         $query = $this->getEntityManager()->createQuery($dql);
+        var_dump($query->getSQL());die;
 
         return $query->execute();
     }
