diff --git a/src/AppBundle/Controller/FortuneController.php b/src/AppBundle/Controller/FortuneController.php
index f4b9e86..66fdf4d 100644
--- a/src/AppBundle/Controller/FortuneController.php
+++ b/src/AppBundle/Controller/FortuneController.php
@@ -15,12 +15,6 @@ class FortuneController extends Controller
      */
     public function homepageAction(Request $request)
     {
-        /** @var EntityManager $em */
-        $em = $this->getDoctrine()->getManager();
-        $filters = $em->getFilters()
-            ->enable('fortune_cookie_discontinued');
-        $filters->setParameter('discontinued', false);
-
         $categoryRepository = $this->getDoctrine()
             ->getManager()
             ->getRepository('AppBundle:Category');
diff --git a/src/AppBundle/EventListener/BeforeRequestListener.php b/src/AppBundle/EventListener/BeforeRequestListener.php
index d9940ea..7efa194 100644
--- a/src/AppBundle/EventListener/BeforeRequestListener.php
+++ b/src/AppBundle/EventListener/BeforeRequestListener.php
@@ -14,6 +14,9 @@ class BeforeRequestListener
 
     public function onKernelRequest(GetResponseEvent $event)
     {
-        // ...
+        $filter = $this->em
+            ->getFilters()
+            ->enable('fortune_cookie_discontinued');
+        $filter->setParameter('discontinued', false);
     }
-}
\ No newline at end of file
+}
