diff --git a/app/config/config.yml b/app/config/config.yml
index 25f6647..b1da2ea 100644
--- a/app/config/config.yml
+++ b/app/config/config.yml
@@ -63,6 +63,8 @@ doctrine:
     orm:
         auto_generate_proxy_classes: "%kernel.debug%"
         auto_mapping: true
+        filters:
+            fortune_cookie_discontinued: AppBundle\Doctrine\DiscontinuedFilter
 
 # Swiftmailer Configuration
 swiftmailer:
diff --git a/src/AppBundle/Controller/FortuneController.php b/src/AppBundle/Controller/FortuneController.php
index 9e11c2b..9f2e087 100644
--- a/src/AppBundle/Controller/FortuneController.php
+++ b/src/AppBundle/Controller/FortuneController.php
@@ -14,6 +14,11 @@ class FortuneController extends Controller
      */
     public function homepageAction(Request $request)
     {
+        $this->getDoctrine()
+            ->getEntityManager()
+            ->getFilters()
+            ->enable('fortune_cookie_discontinued');
+
         $categoryRepository = $this->getDoctrine()
             ->getManager()
             ->getRepository('AppBundle:Category');
diff --git a/src/AppBundle/Doctrine/DiscontinuedFilter.php b/src/AppBundle/Doctrine/DiscontinuedFilter.php
index 80ec6f0..ed7f72e 100644
--- a/src/AppBundle/Doctrine/DiscontinuedFilter.php
+++ b/src/AppBundle/Doctrine/DiscontinuedFilter.php
@@ -17,6 +17,6 @@ class DiscontinuedFilter extends SQLFilter
      */
     public function addFilterConstraint(ClassMetadata $targetEntity, $targetTableAlias)
     {
-        // ...
+        var_dump($targetEntity);die;
     }
 }
