diff --git a/src/AppBundle/Entity/CategoryRepository.php b/src/AppBundle/Entity/CategoryRepository.php
index 5798840..d62f729 100644
--- a/src/AppBundle/Entity/CategoryRepository.php
+++ b/src/AppBundle/Entity/CategoryRepository.php
@@ -24,7 +24,7 @@ class CategoryRepository extends EntityRepository
     public function search($term)
     {
         return $this->createQueryBuilder('cat')
-            ->andWhere('cat.name LIKE :searchTerm')
+            ->andWhere('cat.name LIKE :searchTerm OR cat.iconKey LIKE :searchTerm')
             ->setParameter('searchTerm', '%'.$term.'%')
             ->getQuery()
             ->execute();
